include/arcade/IGame.hpp

```diff
@@ -61,7 +62,8 @@ namespace arcade
         /// This method is called each time the underlying graphics backend is switched.
         ///
         /// @param manager The assets manager.
-        virtual void loadAssets(IAssetManager &manager) = 0;
+        /// @param displaySize The size of the display.
+        virtual void loadAssets(IAssetManager &manager, vec2u &displaySize) = 0;
```

Comment:
Remove the reference, a vec2u is just a pair of two unsigned ints, it is only 64 bits wide, that is the size of pointer/reference.
By passing the vector by reference, you are only adding more overhead.
By the way, you forgot `const`, currently the game can arbitrarily change the display size.